contribs/gmf/src/directives/mobilenav.js

```diff
@@ -50,16 +50,9 @@ gmf.mobileNavDirective = function() {
     controller: 'gmfMobileNavController as navCtrl',
     bindToController: true,
     scope: true,
-    link:
-        /**
-         * @param {angular.Scope} scope Scope.
-         * @param {angular.JQLite} element Element.
-         * @param {angular.Attributes} attrs Atttributes.
-         * @param {gmf.MobileNavController} navCtrl Controller.
-         */
-        function(scope, element, attrs, navCtrl) {
-          navCtrl.init(element);
-        }
+    link(scope, element, attrs, navCtrl) {
```

Comment:
You may want to keep the comments here and below. They simply need to be moved before `link` and their indentation reduced.